spec/js/models/todo_spec.js

```diff
@@ -0,0 +1,37 @@
+define([
+  'models/todo'
+], function (ToDo) {
+  var instance, subject;
+
+  beforeEach(function () {
+    instance = new ToDo();
+    subject = function () {
+      return instance;
+    };
+  });
+
+  describe('Empty ToDo Item', function () {
+    it('will be defined', function () {
+      expect(subject).toBeDefined();
+    });
+  });
+
+  describe('ToDo Item', function () {
+    var secondSubject, secondInstance;
+
+    beforeEach(function () {
+      secondInstance = new ToDo({title: 'Drink beer', completed: false});
+      secondSubject = function () {
```

Comment:
Again testing the framework, the only valid test here is to test that the default values are set, so on creating a new todo the text should be empty and the completed should be false.